api/docker-compose.yml

```diff
@@ -12,7 +12,9 @@ services:
       - default
 
   opensearch-node:
-    image: opensearchproject/opensearch:latest
+    build:
+      context: .
+      dockerfile: Dockerfile_opensearch
```

Comment:
Minor nit:

Could we put that dockerfile in a folder (let's say `opensearch`) and just name it `Dockerfile`? That seems to be the common pattern I've seen for Dockerfiles / sorta follows what we did for mock-oauth configuration as well.